apps/vr-tests-react-components/src/stories/Button/Button.stories.tsx

```diff
@@ -0,0 +1,191 @@
+import * as React from 'react';
+import { Button } from '@fluentui/react-button';
+import { bundleIcon, CalendarMonthFilled, CalendarMonthRegular } from '@fluentui/react-icons';
+import { ComponentMeta } from '@storybook/react';
+import { getStoryVariant, withScreenerSteps, DARK_MODE, HIGH_CONTRAST, RTL } from '../../utilities';
+import { buttonId, steps, useStyles } from './utils';
+
+const CalendarMonth = bundleIcon(CalendarMonthFilled, CalendarMonthRegular);
+
+export default {
+  title: 'Button Converged',
+  Component: Button,
+  decorators: [story => withScreenerSteps({ story, steps })],
+} as ComponentMeta<typeof Button>;
+
+export const Default = () => <Button id={buttonId}>Hello, world</Button>;
+
+export const DefaultRTL = getStoryVariant(Default, RTL);
+export const DefaultDarkMode = getStoryVariant(Default, DARK_MODE);
+export const DefaultHighContrast = getStoryVariant(Default, HIGH_CONTRAST);
+
+export const AsAnAnchor = () => (
+  <Button id={buttonId} as="a" href="#">
+    Hello, world
+  </Button>
+);
+
+AsAnAnchor.storyName = 'As an anchor';
+
+export const Circular = () => (
```

Comment:
Would it be cleaner to have a single function up to that returned the button with props

```
const ButtonStory = (props, variant) => getStoryVariant(<Button {...props}/>, variant)

export Circular = ButtonStory({id: ButtonId, shape: 'circular', children: 'Hello World'});
export CircularRTL = ButtonStory({id: ButtonId, shape: 'circular', children: 'Hello World'}, RTL);
```

Something like this at least. Avoid all the repetition of the Button tags, keep everything to 1 line, easier to parse the file

Reply:
This is mostly codemod generated code plus the manual addition of the `getStoryVariant` stories so I wanted to keep it simple. Will certainly keep this feedback in mind when work to incorporate interaction tests with these are started in a future PR :)